app/event/send-acknowledgement-event.js

```diff
@@ -0,0 +1,12 @@
+const sendInvalidBankDetailsEvent = require('./send-invalid-bank-details-event')
+const sendProcessingAckErrorEvent = require('./send-processing-ack-error-event')
+
+const sendAcknowledgementEvent = async (holdCategoryName, acknowledgement, frn) => {
+  if (holdCategoryName === 'Bank account anomaly') {
+    await sendInvalidBankDetailsEvent(frn)
```

Comment:
Would consider renaming this to `sendProcessingAckInvalidBankDetailsErrorEvent`.

Makes it clear that this error event being sent off comes from processing an acknowledgement event specifically about invalid bank details.

Reply:
Which name should be considered renaming exactly? I'm not sure which one you mean

Reply:
Line 6 function call